docs/source/guides/feature_selection.ipynb

```diff
@@ -0,0 +1,305 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ft.selection.remove_highly_null_features(fm)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that calling `remove_highly_null_features` didn't remove every column that contains a null value. By default, we only remove columns where the percentage of null values is above 95%. If we want to lower that threshold, we can set the `pct_null_threshold` paramter ourselves."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "remove_highly_null_features(fm, pct_null_threshold=.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we're left with a feature matrix containing mostly populated data!\n",
+    "\n",
+    "## Remove Single Value Features\n",
+    "\n",
+    "Another situation we might run into is one where our calculated features don't have any variance. In those cases, we are likely to want to remove the uninteresting columns. For that, we use `remove_single_value_features`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm, features = ft.dfs(entityset=es,\n",
+    "                          target_entity=\"first\",\n",
+    "                          trans_primitives=['is_null'],\n",
+    "                          agg_primitives=[],\n",
+    "                          max_depth=2)\n",
+    "fm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The example of using `IsNull` as a primitive highlights a case where many columns all have the same value. Lets remove them:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "remove_single_value_features(fm)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that we've actually lost two of the three columns with null values because, with the function used as it is above, null values are not considered in whether a column has only one unique value. If we'd like to consider `NaN` its own value, we can set `count_nan_as_value` to `True` and we'll see `quarter_null` back in the matrix."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "remove_single_value_features(fm, count_nan_as_value=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Remove Highly Correlated Features\n",
+    "\n",
+    "The last feature selection function we have allows us to remove columns that would likely be redundant to the model we're attempting to build by considering the correlation between pairs of calculated features.\n",
```

Comment:
To help avoid confusion, let's use columns to refer to columns in the entity/dataset and use features to for the columns of the feature matrix

Reply:
Ah, that makes sense--we just had a similar conversation in woodwork about columns vs DataColumns. Changed a bunch of places where `columns` was being used